secure/json-rpc/src/error.rs

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use hex::FromHexError;
+use std::io;
+use thiserror::Error;
+
+#[derive(Debug, Error, PartialEq)]
+pub enum Error {
+    #[error("JSON RPC call {0} returned a custom internal error: {0}")]
+    InternalRPCError(String, String),
```

Comment:
do we benefit by leaking which call made the error?

Reply:
I was wanting to make the error message as helpful as possible, i.e., without needing to look at the backtrace, but I guess as long as the backtrace is there, that's fine too!